pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -60,14 +61,11 @@ type ActualStateOfWorld interface {
 	// the specified volume, the node is added.
 	AddVolumeNode(logger klog.Logger, uniqueName v1.UniqueVolumeName, volumeSpec *volume.Spec, nodeName types.NodeName, devicePath string, attached bool) (v1.UniqueVolumeName, error)
 
-	// SetVolumeMountedByNode sets the MountedByNode value for the given volume
-	// and node. When set to true the mounted parameter indicates the volume
+	// SetVolumesMountedByNode sets all the volumes mounted by the given node.
```

Comment:
Please add a note it tracks non-attachable volumes too. "All" is IMHO not explicit enough.